### ISSUE TITLE
Assigned parts

### ISSUE THREAD

**Issue by roanraina** [xFiveRivers/DSCI_573_Lab4_Group_Project#1]
TO DO: 
- [x] 1. Pick your problem and explain the prediction problem @xFiveRivers 
- [x] 2. Data splitting @roanraina 
- [x] 3. EDA @roanraina 
- [ ] 4. Feature engineering
- [x] 5. Preprocessing and transformation @xFiveRivers 
- [x] 6. Baseline Model @roanraina 
     - Dummy classifier
- [x] 7. Linear model @roanraina 
     - Ridge
- [x] 8. Different models
  - [x] RandomForest @LisaSeq
  - [x] SVC @mikeguron
  - [x] LightGVM @xFiveRivers 
- [ ] 9. Feature selection
- [x] 10. Hyperparameter optimization
  - [x] RandomForest  @LisaSeq
  - [x] SVC @mikeguron
  - [x] LightGVM @xFiveRivers 
- [x] 11. Interpretation and feature importance @mikeguron
- [x] 12. Results on the test set @mikeguron
- [x] 13. Summary of Results @LisaSeq @xFiveRivers 
- [ ] 14. Create a data analysis pipeline
- [ ] 15. Take away from the course

**Comment by roanraina** [comment]
For part 7) `Ridge` is not a linear classifier (it's a linear regressor) [[look here!](https://scikit-learn.org/stable/modules/classes.html#module-sklearn.linear_model)] ... switched to `LogisticRegression` instead.

**Comment by mikeguron** [comment]
For part 10) updated all models to use "f1" as the scoring metric, instead of the default "accuracy" since that is what was agreed upon in part 1)

**Comment by mikeguron** [comment]
Accidently closed issue, re-opened now

**Comment by xFiveRivers** [comment]
closing issue, submitted
